lms/static/lms/js/require-config.js

```diff
@@ -75,7 +75,9 @@
             "catch": "js/vendor/ova/catch/js/catch",
             "handlebars": "js/vendor/ova/catch/js/handlebars-1.1.2",
             "tinymce": "js/vendor/tinymce/js/tinymce/tinymce.full.min",
-            "jquery.tinymce": "js/vendor/tinymce/js/tinymce/jquery.tinymce.min"
+            "jquery.tinymce": "js/vendor/tinymce/js/tinymce/jquery.tinymce.min",
+            'slick.grid': 'xmodule_js/common_static/js/vendor/slick.grid',
+            'slick.core': 'xmodule_js/common_static/js/vendor/slick.core'
             // end of files needed by OVA
```

Comment:
Is this comment accurate? Are these files needed by OVA? If it's inaccurate, please correct this comment, or delete it.